### PR TITLE
Set up FAQ marketing link

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd 

corresponding PRs:
https://github.com/Stanford-Online/edx-platform/pull/753
https://github.com/Stanford-Online/openedx-themes/pull/11

### CODE REVIEW

**Comment by caesar2164** [comment]
there you go

File: stanford/cme/group_vars/all [hunk 1]
```diff
@@ -27,6 +27,7 @@ EDXAPP_FEATURES_USE_CME_REGISTRATION: true
 EDXAPP_LMS_NGINX_PORT: 80
 EDXAPP_MKTG_URL_LINK_MAP:
   COURSES: 'https://med.stanford.edu/cme/learning-opportunities/online.html'
+  FAQ: 'https://stanfordonline.zendesk.com/hc/en-us'
```
Comment (stvstnfrd) [inline stanford/cme/group_vars/all]: Should this be `https://support.zoho.com/portal/stanfordonlinecmesupport/home`?

Comment (caesar2164) [reply stanford/cme/group_vars/all]: yes it should.

**Comment by stvstnfrd** [comment]
@caesar2164 squash first

**Pull request merged**